index.js

```diff
@@ -27,14 +27,11 @@ var new_client = function(api_key, config) {
   client.proxy_host = config.proxy_host;
   client.proxy_port = config.proxy_port;
   client.proxy_auth = config.proxy_auth;
+  client.proxy_scheme = config.proxy_scheme;
 
   // Initialize global tunnel if proxy options are set
-  if (client.proxy_host && client.proxy_port) {
-    global_tunnel.initialize({
-      host: client.proxy_host,
-      port: client.proxy_port,
-      proxyAuth: client.proxy_auth      
-    });
+  if (client.proxy_host && client.proxy_port ) {
+    client.proxy_agent = create_proxy_agent(config);
```

Comment:
Identifier 'proxy_agent' is not in camel case.<br>Identifier 'create_proxy_agent' is not in camel case.